package.json

```diff
@@ -15,58 +15,58 @@
     "cryptojs": ">= 2.5.3",
     "d3": "3.5.16",
     "envify": "^4.1.0",
-    "file-saver": "1.3.3",
+    "file-saver": "^1.3.8",
     "graceful-fs": "4.1.11",
     "js-yaml": "https://registry.npmjs.org/js-yaml/-/js-yaml-3.9.1.tgz",
-    "marked": "^0.3.9",
+    "marked": "^0.3.19",
     "mixwith": "0.1.1",
-    "prism-languages": "0.3.3",
-    "prismjs": "1.6.0",
-    "prop-types": "15.6.0",
-    "query-string": "^5.0.1",
-    "react": "^16.2.0",
-    "react-click-outside": "^2.3.1",
-    "react-dnd": "2.5.4",
-    "react-dnd-html5-backend": "2.5.4",
-    "react-dom": "^16.2.0",
+    "prism-languages": "^0.4.0",
+    "prismjs": "^1.14.0",
+    "prop-types": "^15.6.1",
+    "query-string": "^6.0.0",
+    "react": "^16.3.2",
+    "react-click-outside": "^3.0.1",
+    "react-dnd": "^2.6.0",
+    "react-dnd-html5-backend": "^2.6.0",
+    "react-dom": "^16.3.2",
     "shapeup": "^0.2.5",
     "xterm": "^2.9.2",
     "yui": "3.11.0",
     "zip": "^1.2.0"
   },
   "devDependencies": {
-    "babel-cli": "6.24.1",
+    "babel-cli": "^6.26.0",
```

Comment:
All these updates with `^` are now non-frozen versions. Is that OK?

Reply:
Yeah because we lock it in the package-lock.json so I didn't feel it necessary to remove them.

Reply:
Sure thing!